Template_Errori_Dati_Mancanti_2.txt

```diff
@@ -0,0 +1,4 @@
+2020-6-01 | 14:22 | 18:24 | Coop | Lorem ipsum G

```

Comment:
Da titolo del file sembrerebbe manchi qualche dato, in realtà ci sono tutti

Reply:
Quindi si dovrebbe cancellare qualche dato per usare il file come Test del Template errato

Reply:
Quando stavo togliendo le <> ho visto che mancava un dato e istintivamente credo di averlo rimesso, ho talto un dato, penso che gli altri vadano bene tutti